### ISSUE TITLE
MA0026 throws ArgumentOutOfRangeException (AD0001) on an unterminated block comment

### ISSUE THREAD

**Issue by meziantou** [meziantou/Meziantou.Analyzer#1328]
`FixToDoAnalyzer` slices multi-line comment trivia with `nodeText[2..^2]`, which assumes the trivia is at least 4 characters. For an unterminated comment Roslyn still produces a `MultiLineCommentTrivia` over the raw text.

https://github.com/meziantou/Meziantou.Analyzer/blob/main/src/Meziantou.Analyzer/Rules/FixToDoAnalyzer.cs#L41-L44

### Repro

Each of these sources — `class C { }` followed by `/*`, or `/*a`, or `/*/` — produces:

```
AD0001: Analyzer 'Meziantou.Analyzer.Rules.FixToDoAnalyzer' threw an exception of type
'System.ArgumentOutOfRangeException' with message 'Specified argument was out of the range of valid values.'
```

`/*ab` (length 4) is fine.

### Why it matters

MA0026 is enabled by default at Warning, and the trigger is the ordinary act of typing `/*` at the end of a file — every keystroke until the closing `*/` produces an analyzer crash in the IDE. On a command-line build of a file that genuinely ends with `/*`, the rule is silently disabled for the whole compilation.

This is only reachable from non-compiling source, which is why the suite does not catch it (filed separately).

### Suggested fix

Guard the slice: remove the trailing `*/` only when the text is at least 4 characters **and** actually ends with `*/` (the trivia is not guaranteed to be terminated). The single-line branch just above assumes length >= 2, which does hold for `//`.

---
<sub>Found during a whole-project review. Reproduced against `Meziantou.Analyzer.Test.roslyn5.9.csproj` at commit 22602bed.</sub>
